### PR TITLE
Adding and renaming cluster repos to the cluster management section of the app

### PR DESCRIPTION
**Pull request by codyrancher**
This allows restricted admins to manage the repos even if they don't have access to the local cluster.

https://github.com/rancher/dashboard/issues/3995]

Screenshot from 2021-09-14 11-21-03
Screenshot from 2021-09-14 11-21-22
Screenshot from 2021-09-14 11-21-28

I verified:

1. A restricted admin has access to cluster repos under cluster management
2. That the breadcrumbs at the top of the page as well as the cancel buttons still work for both local and cluster management variants.

### CODE REVIEW

File: assets/translations/en-us.yaml [hunk 1]
```diff
@@ -4707,7 +4707,7 @@ typeDescription:
   monitoring.coreos.com.prometheus: A Prometheus server is a Prometheus deployment whose scrape configuration and rules are determined by selected ServiceMonitors, PodMonitors, and PrometheusRules and whose alerts will be sent to all selected Alertmanagers with the custom resource's configuration.
   monitoring.coreos.com.alertmanager: An alert manager is deployment whose configuration will be specified by a secret in the same namespace, which determines which alerts should go to which receiver.
   node: The base Kubernetes Node resource represents a virtual or physical machine which hosts deployments. To manage the machine lifecyle, if available, go to Cluster Management.   
-  catalog.cattle.io.clusterrepo: 'A chart repository is a Helm repository or {vendor} git based application catalog. It provides the list of available charts in the cluster.'
+  catalog.cattle.io.clusterrepo: 'A cluster repository is a Helm repository, {vendor} git based application catalog or a rke template repository. It provides the list of available charts and templates in the clusters.'
```
Comment (nwmac) [inline assets/translations/en-us.yaml:4710]: Can we have two messages - only repositories in the local cluster can contain cluster templates.

For the local cluster, user the new message, for non-local clusters, use the old message.

**Review by nwmac — CHANGES_REQUESTED** [review]

File: assets/translations/en-us.yaml [hunk 2]
```diff
@@ -5248,7 +5248,8 @@ typeDescription:
...
   monitoring.coreos.com.alertmanager: An alert manager is deployment whose configuration will be specified by a secret in the same namespace, which determines which alerts should go to which receiver.
   node: The base Kubernetes Node resource represents a virtual or physical machine which hosts deployments. To manage the machine lifecyle, if available, go to Cluster Management.   
-  catalog.cattle.io.clusterrepo: 'A chart repository is a Helm repository or {vendor} git based application catalog. It provides the list of available charts in the cluster.'
+  catalog.cattle.io.clusterrepo: 'A repository is a Helm repository or {vendor} git based application catalog. It provides the list of available charts in the cluster.'
+  catalog.cattle.io.clusterrepo.local: 'A repository is a Helm repository, {vendor} git based application catalog or a rke template repository. It provides a list of available charts and templates.'
```
Comment (richard-cox) [inline assets/translations/en-us.yaml:5252]: should rke be capitalised?

Comment (richard-cox) [reply assets/translations/en-us.yaml:5252]: Strictly speaking the rke template repository is one of the two previous ones, which is offered to the user in a radio button in the create flow.
Maybe something like...

`A repository is either a Helm or {vendor} git based catalog. It provides a list of available Helm charts and templates, and RKE templates.`

for non-local...
`A repository is either a Helm or {vendor} git based catalog. It provides a list of available charts and templates.`

File: list/catalog.cattle.io.clusterrepo.vue [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      required: true,
+    },
+  },
+
+  computed: {
+    ...mapGetters(['currentCluster']),
+    hideDescriptions: mapPref(HIDE_DESC),
+
+    typeDescriptionKey() {
+      return this.currentCluster.isLocal ? 'typeDescription."catalog.cattle.io.clusterrepo.local"' : 'typeDescription."catalog.cattle.io.clusterrepo"';
```
Comment (richard-cox) [inline list/catalog.cattle.io.clusterrepo.vue:38]: When signing in as a standard user and going to the management repo list i get a `Cannot read property 'isLocal' of undefined` error. In this context, it's safe to assume if there's no currentCluster then we're seeing catalog resources from that local cluster

**Review by richard-cox — CHANGES_REQUESTED** [review]

**Comment by nwmac** [comment]
@codyrancher I've made some tweaks to this. Just going to tweak the banner label.

**Review by richard-cox — APPROVED** [review]

**Pull request merged**